easybuild/easyconfigs/i/iomkl/iomkl-2016.03b.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = "Toolchain"
+
+name = 'iomkl'
+version = '2016.03b'
```

Comment:
When I did it  I understood the names were derived from the intel version included (16.3.210).  I first used GCC 4.9.3 as the underlying compiler but you sort of asked for 5.4.0 (see your email).  To avoid interference on our local system, version with GCC 5.4.0 is 2016.03b.  2016.03 only lives on our system ;)